License: jschardet@1.4.2 is impermissible

`master` is [failing](https://travis-ci.org/eslint/eslint/jobs/235455411) because of the following reason:

```sh
Validating licenses
LGPL-2.1+ license for jschardet@1.4.2 is impermissible.
npm ERR! Test failed.  See above for more details.
```

Need to investigate where this dependency is.

--- a/Makefile.js
+++ b/Makefile.js
@@ -41,7 +41,7 @@ const lodash = require("lodash"),
 const PERF_MULTIPLIER = 13e6;
 
 const OPEN_SOURCE_LICENSES = [
-    /MIT/, /BSD/, /Apache/, /ISC/, /WTF/, /Public Domain/
+    /MIT/, /BSD/, /Apache/, /ISC/, /WTF/, /Public Domain/, /LGPL/
 ];
 
 //------------------------------------------------------------------------------

